index.js

```diff
@@ -1,16 +1,12 @@
-import iopipe from '@iopipe/iopipe'
-
-//import aws from 'aws-sdk'
+const iopipe = require('@iopipe/iopipe');
 const aws = require('aws-sdk');
 const jsonwebtoken = require('jsonwebtoken');
 const fs = require('fs');
 const keys = require('./keys');
 const crypto = require('crypto');
 
-//const iopipe_profiler = require('@iopipe/profiler');
 const IOpipe = iopipe({
   token:"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJhNDAyZTQzNy0wNzIyLTQ0ZDktOGUyNy1jMGFjMjc2MzgxZTQiLCJqdGkiOiJmYTQzMDQxMi0zYThkLTQ5MzgtODA4My00YWIxODRjNmZmZjEiLCJpYXQiOjE1MDAzODgzMjgsImlzcyI6Imh0dHBzOi8vaW9waXBlLmNvbSIsImF1ZCI6Imh0dHBzOi8vaW9waXBlLmNvbSxodHRwczovL21ldHJpY3MtYXBpLmlvcGlwZS5jb20vZXZlbnQvLGh0dHBzOi8vZ3JhcGhxbC5pb3BpcGUuY29tIn0.Iez7L1pRsC1gk50H6-Qh99ZaduFfCixAPxgkfPmpElI",
```

Comment:
we need to remove this token before release and include docs on how to add the token and signup for IOpipe